src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/package.json

```diff
@@ -114,53 +114,13 @@
-                    "type": "object",
-                    "required": [],
-                    "default": {},
-                    "properties": {
-                      "exceptions": {
-                        "type": "boolean",
-                        "description": "Optional flag to determine whether exception messages should be logged to the output window.",
-                        "default": true
-                      },
-                      "moduleLoad": {
-                        "type": "boolean",
-                        "description": "Optional flag to determine whether module load events should be logged to the output window.",
-                        "default": true
-                      },
-                      "programOutput": {
-                        "type": "boolean",
-                        "description": "Optional flag to determine whether program output should be logged to the output window when not using an external console.",
-                        "default": true
-                      },
-                      "engineLogging": {
-                        "type": "boolean",
-                        "description": "Optional flag to determine whether diagnostic engine logs should be logged to the output window.",
-                        "default": false
-                      },
-                      "browserStdOut": {
-                        "type": "boolean",
-                        "description": "Optional flag to determine if stdout text from the launching the web browser should be logged to the output window.",
-                        "default": true
-                      },
-                      "elapsedTiming": {
-                        "type": "boolean",
-                        "description": "If true, engine logging will include `adapterElapsedTime` and `engineElapsedTime` properties to indicate the amount of time, in microseconds, that a request took.",
-                        "default": false
-                      },
-                      "threadExit": {
-                        "type": "boolean",
-                        "description": "Controls if a message is logged when a thread in the target process exits. Default: `false`.",
-                        "default": false
-                      },
-                      "processExit": {
-                        "type": "boolean",
-                        "description": "Controls if a message is logged when the target process exits, or debugging is stopped. Default: `true`.",
-                        "default": true
-                      }
-                    }
+                  "dotNetConfig": {
+                    "description": "Options passed to ",
+                    "type": "object"
+                  },
+                  "browserConfig": {
+                    "description": "Options based",
+                    "type": "object"
```

Comment:
Is it just an object because it's a generic key=>value dictionary with no structure?

Reply:
It does have a structure. It should only contain the properties that you would pass to the JS debugger (or .NET debugger in the case above). I'll add docs to clarify this.

We could copy/paste the entire config setup for the .NET/JS debuggers here but that didn't seem sustainable.

Reply:
Hmmmmm. Good point. I'll strike a balance by including the options that users are more likely to use here.